auth0_to_hubspot/README.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Auth0 to HubSpot
```

Comment:
What do you think about "Copy Auth0 Users to HubSpot"? Still short, but clearer.

Reply:
I couldn't think of a better name, so I think your idea is great.